docs/content/configuration/csi_ug_config_create_secret_topology.md

```diff
@@ -0,0 +1,46 @@
+# Creating a Secret with topology awareness
+
+Create an array secret YAML file in order to define the storage credentials (username and password) and address. Use this information for creating a Secret that is topology aware.
+
+**Note:** If you are not using the CSI Topology feature, follow the steps in [Creating a Secret](csi_ug_config_create_secret.md).
+
+Use the Secret file to connect a worker node to a storage system.
```

Comment:
that can be sound like by using that secret we are automatically connecting the actual worker to the storage..

Reply:
```suggestion
Use the Secret file to allow connecting to a storage system.
```

Reply:
isn't it the purpose of every secret? 
maybe something more like: 

> Use the Secret file to reflect the connection mapping of the worker nodes to a storage system.

what do you think?

Reply:
@ArbelNathan  @oriyarde  I am actually not sure why we have this sentence. it's not in the "original" secret file. Can I just remove it all together? or is it needed for topology and I should write it as was suggested:
"Use the Secret file to reflect the connection mapping of the worker nodes to a storage system."

Reply:
I would remove it altogether